src/components/ProgressBar/ProgressBar.scss

```diff
@@ -0,0 +1,19 @@
+@import 'styles/font-mixins';
+
+.ProgressBar {
+  &__bar {
+    background-color: var(--color-gray-100);
+    border-radius: 100px;
+    height: 8px;
+    margin-bottom: 8px;
```

Comment:
Not an issue, but it's a good practice to avoid adding margins on components. Let the container handle it.